benchmark/benchmark.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -e
+trap cleanup EXIT
+
+export BENCHMARK_STACK_NAME=${BENCHMARK_STACK_NAME:-"powertools-benchmark"}
+
+function cleanup {
+    echo "Cleaning up stack..."
+    aws cloudformation delete-stack --stack-name $BENCHMARK_STACK_NAME
+}
+
+function run_function {
+    # Update function to force a cold start
+    aws lambda update-function-configuration --function-name $1 --memory-size 256 >/dev/null
```

Comment:
can we add 512 too? It's quite common to see 256, 512, and 1024. can have skewed results

Reply:
This doesn't really test at 256, it's just a trick to force a cold start. 😄